packages/perseus/src/types.ts

```diff
@@ -569,9 +538,6 @@ export type WidgetExports<
      * This key defaults to `{major: 0, minor: 0}` if not provided.
      */
     version?: Version;
-    supportedAlignments?: ReadonlyArray<Alignment>;
-    defaultAlignment?: Alignment;
-    getDefaultAlignment?: () => Alignment;
```

Comment:
I got rid of `getDefaultAlignment`. It didn't seem like any of the components used it?